packages/platforms/browser/lib/auto-instrumentation/page-load-phase-spans.ts

```diff
@@ -0,0 +1,73 @@
+import type { SpanFactory } from '@bugsnag/core-performance'
+import { type PerformanceWithTiming } from '../on-settle/load-event-end-settler'
+
+function shouldOmitSpan (startTime: number, endTime: number) {
+  return startTime === 0 && endTime === 0
+}
```

Comment:
This should handle `undefined` since some of these aren't present on Safari <15, e.g.:

```suggestion
function shouldOmitSpan (startTime?: number, endTime?: number) : boolean {
  return !startTime && !endTime
}
```